### ISSUE TITLE
”Since the input of the I2V DiT model has more channels than T2V (i.e., 2 × c + s v.s. c), an additional projection layer is employed“

### ISSUE THREAD

**Issue by jialiangZ** [Wan-Video/Wan2.1#404]
Great work, in the first paragraph of Model Design in 5.1.1 of the paper, the input of I2V contains noise, condition latent and mask, these three variables are concatenated in the code directly in the channel dimension, in that case, the total number of channels is 2 × c + s, but the code doesn't There is no projection layer mentioned in the paper, what is used here? Is it simply nn.Linear? [x = [torch.cat([u, v], dim=0) for u, v in zip(x, y)]](https://github.com/Wan-Video/Wan2.1/blob/c709fcf0e7771e0b7f5e7a72d486b5358dc6e722/wan/modules/model.py#L530)

**Comment by jialiangZ** [comment]
I found it, just changed the in to 36
